tweak abstract and intro (#13)
danwing/signaling-use-cases: draft-rwbr-tsvwg-signaling-use-cases.md

adaptive transmission or session migration), or through cooperation of both the host
and the network.

> This document lists uses-cases demonstrating the need for a mechanism

Reviewer: ```suggestion
This document lists uses cases demonstrating the need for a mechanism
```

to share metadata about flows between a receiving host and its network

> to enable different traffic treatment for packets sent to the

Reviewer: ```suggestion
to enable differentiated traffic treatment for packets sent to the
```

performance constraints usually exist with a "B" (for Bottleneck).
Other network bottlenecks may be experienced in other segments not shown
in the figure, such as interconnection links or the infrastructure that hosts the service (e.g.,
flash crowds).  When a bottleneck exists temporarily, the network has no choice but

> to discard or delay packets -- which can harm certain flows.  In this

Reviewer: ```suggestion
to discard or delay packets -- which can harm certain flows and thus lead to suboptimal perceived experience.  In this
```

> paper, this is termed 'reactive policy'.

Reviewer: ```suggestion
document, this is termed 'reactive policy'.
```

with flow metadata. Such augmentation allows those network elements
to make autonomous decisions to prioritize, delay, or drop packets,
especially when performing reactive resource management. Absent such metadata,
these network elements have no guidance and treat packets indiscriminately.

There are several challenges with this metadata augmentation:

* for hosts:
  * which data to share without privacy breach or lowering confidentiality.

* for network elements:
  * Which metadata to honor
  * Tradeoff between the extra cost (including processing) versus benefits
  * Operational impacts

> Configured cooperation between an ISP and content providers can allows

Reviewer: ```suggestion
Configured cooperation between an ISP and content providers (via contractual agreements, typically) can allow
```

metadata signals augmenting packets to be honored by the ISP.  This
cooperation has historically involved examination of server IP
address, TLS SNI, AS number, or heuristics to identify flows.

> However, this cooperation favors large Internet service providers and

Reviewer: ```suggestion
However, this cooperation might have scalability issues (e.g., ISPs to establish and maintain agreements with a large number of content providers), operational implications as the network configuration will rely on service-specific (e.g., risk of frozen or stale configuration), mismatch between the expected service level from users and the delivered one, etc. 
```

> favors large content providers.  Smaller ISPs, small content

Reviewer: ```suggestion
Moreover, smaller ISPs, small content
```

providers, and new content providers are harmed.

> A more egalitarian approach provides the same benefit ot parties --

Reviewer: ```suggestion
A more egalitarian approach provides the same benefit to parties --
```

large and small -- and also provide richer signaling to further
improve user experience and metadata interoperability. This allows all

> parties to become part of the "Internet fast lane".

Reviewer: ```suggestion
parties to become part of the "Internet fast lane". Also, this approach improves the Internet for end users per {{?RFC8890}}.
```

Rather than relying on configured cooperation between ISPs and

> content providers, this paper shows use-cases where the client

Reviewer: ```suggestion
content providers, this document shows use cases where the client
```

tells its ISP the importance of packets it is receiving.  This
provides several benefits described in later sections.

There is already some consensus that applications can benefit by collaborative signaling
the network ({{?IAB=RFC9419}}, {{ATIS}}).  This document provides

> use-cases to further detail the need of such signaling and highlights

Reviewer: ```suggestion
use cases to further detail the need of such signaling and highlights
```
